prog/vm/github_runner.rb

```diff
@@ -93,28 +93,21 @@ def before_run
     # We need to reconnect machine to load environment variables again.
     vm.sshable.invalidate_cache_entry
 
+    # We placed the script in the "/usr/local/share/" directory while generation
+    # the golden image. However, it needs to be moved to a home directory because
+    # the runner creates some configuration files at the root level. The "runner"
```

Comment:
I didn't understand where does runner creates configuration files. Did you mean at the working directory?

Reply:
Yeah. It creates at the script location ("/home/runner/actions-runner").